src/boot/stylesheets/actions.scss

```diff
@@ -75,34 +75,44 @@
 	&:only-child {
 		padding: 0 18px;
 	}
-
-	a {
-		outline: none;
-	}
-
-	a.inactive-action:hover {
-		color: $orange-jazzy;
-	}
-
-	a.active-action:hover {
-		color: $gray-text-min;
-	}
 }
 
 .wpnc__main div.wpnc__action-link {
+	cursor: pointer;
 	text-align: center;
 	font-size: 12px;
+	height: 55px;
```

Comment:
Removed and no longer applicable with the refactor in 3cdefcff4dab7f3d899bac5b1823cf8c5c280806 and 00064b2c268b07d248188746b5144a6c91d198eb.